test/cmd/builds.sh

```diff
@@ -113,7 +113,7 @@ os::test::junit::declare_suite_end
 
 os::test::junit::declare_suite_start "cmd/builds/config"
 os::cmd::expect_success_and_text 'oc describe buildConfigs ruby-sample-build' "Webhook GitHub.+${url}/oapi/v1/namespaces/${project}/buildconfigs/ruby-sample-build/webhooks/secret101/github"
-os::cmd::expect_success_and_text 'oc describe buildConfigs ruby-sample-build' "Webhook Generic.+${url}/oapi/v1/namespaces/${project}/buildconfigs/ruby-sample-build/webhooks/secret101/generic"
+os::cmd::expect_success_and_text 'oc describe buildConfigs ruby-sample-build' "Webhook Generic.+${url}/oapi/v1/namespaces/${project}/buildconfigs/ruby-sample-build/webhooks/secret101/generic/false"
```

Comment:
test case reverted ... though i had to tweak it now that the `oc describe ...` output is on multiple lines per the proposal from @bparees